### PR TITLE
Add RedHat 8 support

### LINKED ISSUE #643: Add support for CentOS/RHEL 8
Currently there is no support for CentOS 8. Could this be added to the module?

### PR DESCRIPTION
**Pull request by tjikkun**
#### Pull Request (PR) description
This PR adds support for EL-8 hosts. It makes sure php-fpm is used there and adds EL-8 to the tests

#### This Pull Request (PR) fixes the following issues
Fixes #643
Closes #687

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -59,37 +59,43 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7"
+        "7",
+	"8"
```
Comment (bastelfreak) [inline metadata.json:63]: please align the new entries so they have the same indentation.

**Review by root-expert — APPROVED** [review]
LGTM, although we don't acceptance tests due to PID file workaround ☹️

File: metadata.json (hunk 1 above)
Comment (tjikkun) [reply metadata.json]: Hum, sorry about that, vim preferred tabs it seems

**Comment by tjikkun** [comment]
I don't know anything about that pid-file workaround, is that something I can fix? Do you have more info on that subject?

**Review by bastelfreak — APPROVED** [review]

**Comment by tjikkun** [comment]
Ah, found it, https://github.com/voxpupuli/puppet_metadata
Unrelated to this MR indeed

**Comment by root-expert** [comment]
Thanks for the contribution!

**Pull request merged**